haddock/defaults.py

```diff
@@ -37,57 +38,108 @@ class Default:
-        "trans_vector_16": data_path / "toppar/initial_positions/trans_vector_16",
-        "trans_vector_17": data_path / "toppar/initial_positions/trans_vector_17",
-        "trans_vector_18": data_path / "toppar/initial_positions/trans_vector_18",
-        "trans_vector_19": data_path / "toppar/initial_positions/trans_vector_19",
-        "trans_vector_20": data_path / "toppar/initial_positions/trans_vector_20",
-        "trans_vector_21": data_path / "toppar/initial_positions/trans_vector_21",
-        "trans_vector_22": data_path / "toppar/initial_positions/trans_vector_22",
-        "trans_vector_23": data_path / "toppar/initial_positions/trans_vector_23",
-        "trans_vector_24": data_path / "toppar/initial_positions/trans_vector_24",
-        "trans_vector_25": data_path / "toppar/initial_positions/trans_vector_25",
-        "trans_vector_26": data_path / "toppar/initial_positions/trans_vector_26",
-        "trans_vector_27": data_path / "toppar/initial_positions/trans_vector_27",
-        "trans_vector_28": data_path / "toppar/initial_positions/trans_vector_28",
-        "trans_vector_29": data_path / "toppar/initial_positions/trans_vector_29",
-        "trans_vector_30": data_path / "toppar/initial_positions/trans_vector_30",
-        "trans_vector_31": data_path / "toppar/initial_positions/trans_vector_31",
-        "trans_vector_32": data_path / "toppar/initial_positions/trans_vector_32",
-        "trans_vector_33": data_path / "toppar/initial_positions/trans_vector_33",
-        "trans_vector_34": data_path / "toppar/initial_positions/trans_vector_34",
-        "trans_vector_35": data_path / "toppar/initial_positions/trans_vector_35",
-        "trans_vector_36": data_path / "toppar/initial_positions/trans_vector_36",
-        "trans_vector_37": data_path / "toppar/initial_positions/trans_vector_37",
-        "trans_vector_38": data_path / "toppar/initial_positions/trans_vector_38",
-        "trans_vector_39": data_path / "toppar/initial_positions/trans_vector_39",
-        "trans_vector_40": data_path / "toppar/initial_positions/trans_vector_40",
-        "trans_vector_41": data_path / "toppar/initial_positions/trans_vector_41",
-        "trans_vector_42": data_path / "toppar/initial_positions/trans_vector_42",
-        "trans_vector_43": data_path / "toppar/initial_positions/trans_vector_43",
-        "trans_vector_44": data_path / "toppar/initial_positions/trans_vector_44",
-        "trans_vector_45": data_path / "toppar/initial_positions/trans_vector_45",
-        "trans_vector_46": data_path / "toppar/initial_positions/trans_vector_46",
-        "trans_vector_47": data_path / "toppar/initial_positions/trans_vector_47",
-        "trans_vector_48": data_path / "toppar/initial_positions/trans_vector_48",
-        "trans_vector_49": data_path / "toppar/initial_positions/trans_vector_49",
-        "trans_vector_50": data_path / "toppar/initial_positions/trans_vector_50"
+        "trans_vector_0": (data_path /
```

Comment:
This is a classic example where I would not personally follow strict 80-lenght line except if committing a major number of lines is intended in order to inflate statistics.

Reply:
ooh this is great @joaomcteixeira, thanks!